manifests/package.pp

```diff
@@ -49,7 +108,7 @@
       # $pkg_version is passed to Package[sensu] { ensure }.  The Windows MSI
       # provider translates hyphens to dots, e.g. '0.29.0-11' maps to
       # '0.29.0.11' on the system.  This mapping is necessary to converge.
-      $pkg_version = template('sensu/sensu-windows-package-version.erb')
+      $pkg_version = regsubst($::sensu::version, '-', '.')
```

Comment:
same is true for sensu-version.erb. I have created the issue https://github.com/sensu/sensu-puppet/issues/731 to get it changed later on too.